internal/exec/util/file.go

```diff
@@ -136,7 +136,8 @@ func (u Util) WriteLink(s types.Link) error {
 	}
 
 	if s.Hard {
-		return os.Link(s.Target, path)
+		targetPath := u.JoinPath(s.Target)
```

Comment:
Is this Util the correct instance? Or will it always hardlink to the root partition?